test/org/impact2585/frc2016/tests/ArmSystemTest.java

```diff
@@ -0,0 +1,182 @@
+package org.impact2585.frc2016.tests;
+
+import org.impact2585.frc2016.input.InputMethod;
+import org.impact2585.frc2016.systems.ArmSystem;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+
+/**
+ * the unit test for the arm system
+ */
+public class ArmSystemTest {
+	private double topArmSpeed;
+	private double bottomArmSpeed;
+	private double bottomArmForwardValue;
+	private double bottomArmBackwardValue;
+	private boolean topArmForward;
+	private boolean topArmBackward;
+	private TestArmSystem arm;
+	private InputMethod input;
+
+	/**
+	 * sets up the test
+	 */
+	@Before
+	public void setUp() {
+		topArmSpeed = 0;
+		bottomArmSpeed = 0;
+		bottomArmForwardValue = 0;
+		bottomArmBackwardValue = 0;
+		input = new InputTest();
+		arm = new TestArmSystem();
+		arm.setInput(input);		
+	}
+
+	/**
+	 * runs various tests for the arm system
+	 */
+	@Test
+	public void test() {
+		
+		//tests if the arms do not move initially
```

Comment:
I redid the indentation in eclipse with ctrl a and ctrl i, and the spacing
looks the same in the diff as well as the actual project.

On Mon, Feb 15, 2016 at 9:19 PM, Michael Murphey <notifications@github.com>
wrote:

> In test/org/impact2585/frc2016/tests/ArmSystemTest.java
> <https://github.com/Impact2585/FRC2016/pull/28#discussion_r52965385>:
>
> > +		topArmSpeed = 0;
> > +		bottomArmSpeed = 0;
> > +		bottomArmForwardValue = 0;
> > +		bottomArmBackwardValue = 0;
> > +		input = new InputTest();
> > +		arm = new TestArmSystem();
> > +		arm.setInput(input);		
> > +	}
> > +
> > +	/**
> > +	 * runs various tests for the arm system
> > +	 */
> > +	@Test
> > +	public void test() {
> > +		
> > +		//tests if the arms do not move initially
>
> It looks like the lines below use spaces instead of tabs when everywhere
> else uses tabs.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Impact2585/FRC2016/pull/28/files#r52965385>.
>